index.js

```diff
@@ -169,4 +169,30 @@ const isEmptyMapOrSet = x => (is.map(x) || is.set(x)) && x.size === 0;
 
 is.empty = x => !x || isEmptyStringOrArray(x) || isEmptyObject(x) || isEmptyMapOrSet(x);
 
+is.any = function (predicate, values) {
```

Comment:
1. See [Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) for the predicate check.
1. Take the values as an array using `(predicate, ...values) => { ... }` instead of the slice.
1. Use arrow functions instead of an anonymous function. All other methods use this syntax. You have to be consistent with the rest of the code.

basically, I think the code can look like `is.any = (predicate, ...values) => values.some(predicate)`;
of course, you have to validate the `values` is an array using `is.array`.

Reply:
I now see that `...values` is impossible, So as I've said, put a comment stating the it's not possible due to node4 compatibility.

So keep the current logic, but use the array methods like before, and extract a common method to which you can pass the correct method (aka `Array.prototype.some` or `Array.prototype.every`).